.travis.yml

```diff
@@ -8,6 +8,10 @@ jobs:
     - stage: jdk11
       dist: bionic
       jdk: openjdk11
+    - stage: lint jdk11
+      dist: bionic
+      jdk: openjdk11
+      script: ./dev-setup && lein eastwood && lein check
 script: "./ext/travisci/test.sh"
 branches:
```

Comment:
Ah I found it in the Travis log.
```
Branch "main" not included per configuration. 
```
This branches config specifies only `master`